apps/andi/config/config.exs

```diff
@@ -0,0 +1,32 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Mix.Config module.
+#
+# This configuration file is loaded before any dependency and
+# is restricted to this project.
+
+# General application configuration
+use Mix.Config
+
+# Configures the endpoint
+config :andi, AndiWeb.Endpoint,
+  url: [host: "localhost"],
+  secret_key_base: "z7Iv1RcFiPow+/j3QKYyezhVCleXMuNBmrDO130ddUzysadB1stTt+q0JfIrm/q7",
```

Comment:
Is this a good time to remove this secret key from the source control before we migrate it into another (more permanent) repo?

Reply:
I think it's best (for local development if nothing else) to leave this here. I put a comment in saying that it should be overwritten.